test/e2e/positive.go

```diff
@@ -17,16 +17,37 @@
 package e2e
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/fromanirh/deployer/pkg/clientutil"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
please group the imports properly:

stdlib

kube

3rd party

local project imports